File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Mascotas/NewPetRequestDto.ts
export class NewPetRequestDto {
  canil: boolean;       // Indicador si la mascota está en un canil
  idSize: number;       // ID del tamaño (referencia a la tabla de tamaños)
  idType: number;       // ID del tipo (referencia a la tabla de tipos)
  userId: number;       // ID del usuario dueño de la mascota
  name: string;

  constructor(
    canil: boolean,
    idSize: number,
    idType: number,
    userId: number,
    name: string
  ) {
    this.canil = canil;
    this.idSize = idSize;
    this.idType = idType;
    this.userId = userId;
    this.name = name;
  }
}

File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/pet/NewPetRequestDto.java
package ps.com.viajeros.dtos.pet;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class NewPetRequestDto {
    private boolean canil;
    private Long idSize;      // ID del tamaño, por ejemplo, 1 (Chico), 2 (Mediano), etc.
    private Long idType;      // ID del tipo, por ejemplo, 1 (Perro), 2 (Gato), etc.
    private Long userId;      // ID del usuario asociado
    private String name;
}

